webrtc.html

```diff
@@ -2075,6 +1826,314 @@
+                "#event-negotiation">negotiationneeded</a> event at
+                <var>connection</var>.</p>
+              </li>
+            </ol>
+          </dd>
+
+          <dt>attribute EventHandler ontrack</dt>
+
+          <dd>This event handler, of event handler event type <code><a href=
+          "#event-track">track</a></code>, MUST be fired
+          by all objects implementing the <code><a>RTCPeerConnection</a></code>
+          interface. It is called any time a <code>MediaStreamTrack</code> is added
+          by the remote peer. This will be fired only as a result of
+          <code>setRemoteDescription</code>. ontrack happens as early as
+          possible after the <code>setRemoteDescription</code>. This callback
+          does not wait for a given media track to be accepted or rejected via
+          SDP negotiation.</dd>
+
+        </dl>
+
+      </section>
+
+      <section>
+        <h3>RTCRtpSender Interface</h3>
+
+        <p>The <code>RTCRtpSender</code> interface allows an application to control how a given
+        <code>MediaStreamTrack</code> is encoded and transmitted to a remote peer.
+        When attributes on an <code>RTCRtpSender</code> are modified, the encoding is either
+        changed appropriately, or a negotiation is triggered to signal the new encoding
+        parameters to the other side.</p>
+
+        <dl class="idl" title=
+          "interface RTCRtpSender">
+        <dt>readonly attribute MediaStreamTrack track</dt>
+
+        <dd>
+          <p>The <dfn id=
+          "dom-rtpsender-track"><code>RTCRtpSender.track</code></dfn>
+          attribute is the track that is associated with this
+          <code><a>RTCRtpSender</a></code> object.</p>
+        </dd>
+      </section>
+
+      <section>
+        <h3>RTCRtpReceiver Interface</h3>
+
+        <p>The <code>RTCRtpReceiver</code> interface allows an application to control the receipt
+        of a <code>MediaStreamTrack</code>. When attributes on an <code>RTCRtpReceiver</code> are modified, a negotiation is triggered to signal the changes regarding what the application
+        wants to receive to the other side.</p>
+
+        <dl class="idl" title=
+          "interface RTCRtpReceiver">
+        <dt>readonly attribute MediaStreamTrack track</dt>
+
+        <dd>
+          <p>The <dfn id=
+          "dom-rtpreceiver-track"><code>RTCRtpReceiver.track</code></dfn>
+          attribute is the track that is immutably associated with this
+          <code><a>RTCRtpReceiver</a></code> object.</p>
+        </dd>
+      </section>
+
+        <section>
+      <h3>RTCTrackEvent</h3>
+
+      <p>The <code><a href="#event-track">ontrack</a></code>
+      event uses the
+      <code><a>RTCTrackEvent</a></code> interface.</p>
+
+      <p><dfn id="fire-track-event" title="fire track event">Firing an
+      RTCTrackEvent event named <var>e</var></dfn> with an
+      <code><a>MediaStreamTrack</a></code> <var>track</var> means that an event
```

Comment:
The two other "arguments" to the event should be mentioned here to make it consistent with other events.